lib/model/store.dart

```diff
@@ -1218,8 +1248,17 @@ class UpdateMachine {
       if (_disposed) return;
     }
 
-    await store._globalStore._reloadPerAccount(store.accountId);
-    assert(_disposed);
+    try {
+      await store._globalStore._reloadPerAccount(store.accountId);
+    } on AccountNotFoundException {
+      // The account was logged out while we retry to replace the store,
+      // but that's OK as long as other code will be removing it from the UI
+      // (usually by using [routeToRemoveOnLogout]).
+      assert(debugLog('… Event queue not replaced; account logged out.'));
```

Comment:
nit: rather than "replace the store", how about "reload the store", as more directly what we mean, and matched with the name `_reloadPerAccount` in the code in the `try`.

Reply:
This is to match the other `debugLog` lines in the function, such as the ones that can lead to this line:
```dart
        assert(debugLog('Lost event queue for $store.  Replacing…'));
```

```dart
        assert(debugLog('BUG: Unexpected error in event polling: $error\n' // TODO(log)
          'Replacing event queue…'));
```

I think it is appropriate to use either "replace the event queue" or "reload the store" consistently. I feel that "replace the event queue" is less vague here because this attempted fix only makes sense when you relate it to the API layer.

Reply:
Ah yeah; "replace the queue" or "replace the event queue" would be fine for the code comment.